Library/.rubocop.yml

```diff
@@ -183,7 +183,6 @@ Naming/MethodName:
     - '\A(fetch_)?HEAD\?\Z'
 
 # allow those that are standard
-# TODO: try to remove some of these
```

Comment:
I think we can leave this comment as-is and aim to remove more later.

I'm personally of the belief that single/double letter variables almost always hinder readability more than the value they provide through speedy typing.

The only one I can see being ever valuable here is `f` and `pr` because their meanings are more obvious but even `f` has been widely misused to mean `f(ormula or cask)` which hinders readability.

Reply:
I think there's an argument that `pr` as an allowed variable should be upstreamed to core RuboCop given it's so ubiquitous, but I'll try that another day.

Reply:
@issyl0 Yeh, I'm a strong 👍🏻 on "if we can get `Exclude:` and/or `Enabled: false` changes upstreamed then let's do so and otherwise: let's try to move towards a configuration RuboCop supports".